src/PFire.Core/Models/FriendRequestModel.cs

```diff
@@ -2,7 +2,7 @@
 {
     public class FriendRequestModel
     {
-        public int Id { get; set; }
+        public uint Id { get; set; }
```

Comment:
Let's keep these as `int` since it interacts with the database. We may need to add casting in places between our database and the protocol though.